_posts/2021/2021-06-24-finding-our-go-to-video-conferencing-platform.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: Finding our go-to video conferencing platform 
+date: 2021-06-24
+header_image: public/pexels-conference.jpeg
+header_position: center
+header_overlay: true
+category: methods-and-tools
+tags: ["video","teams","meet","bigbluebutton","meetings"]
+authors: ["Borja"]
+about_authors: ["bdelafuente"]
+---
+
+Until a little over a year ago, for most of us, participating in video conferences was limited to a few occasions.
+Sometimes it was necessary to speak with colleagues in a different location or when someone was working from home.
+However, the current situation with COVID-19 required some substantial changes. 
+Now, the majority is working from home but Meetings are still a necessity.
+Maybe, they have become even more important to stay in touch with colleagues. 
+This made video conferencing platforms essential for our everyday work.
+
+## Starting the journey
+
+Some months ago, we've started to use [Microsoft Teams](https://www.microsoft.com/de-de/microsoft-teams/group-chat-software){:target="_blank"} which allows us to hold meetings with more than 70 people, generate different rooms that can be used simultaneously, chat with each other, … 
+However, Microsoft Teams isn't the only option. 
+Today, we use a variety of different video platforms within the company depending on the needs and personal preferences of each team.
+In this blog post, I'll talk about some of these options. comment on their characteristics, and explain why we haven't found or go-to yet.
+
+During the first months of 2020, one application in particular quite quickly gained popularity and users - [ZOOM](https://zoom.us/){:target="_blank"}.
+We also thought about using it but there were some issues. 
+At ePages, everyone can freely decide what operating system they want to work with.
+For some developers, Linux is the best option.
+However, there were some bigger differences regarding the range of functions between Windows/Mac and Linux.
+That's why we discarded it as a viable option. 
+No to mention the [doubts about the security](https://www.youtube.com/watch?v=K7hIrw1BUck){:target="_blank"}. 
+
+One tool that a lot of us use every week and some even every day is [BBB (BigBlueButton)](https://bigbluebutton.org/){:target="_blank"}.
+It is an open-source tool based on Linux for video conferences.
+However, you can also record them and the tool provides a chat.
+Due to the need for a physical server in the office, when we started using it, we had some problems with interruptions and slowly running meetings. 
```

Comment:
Add more information. It's a bit unclear why there were problems because you needed a physical server in the office.

Reply:
If you want to work with BBB you need physically a server, in our case it was in the office, with other plataforms you don't need anything, only connect Internet. Microsoft or google have their own servers.

Reply:
I got that. What I mean is that it isn't clear why there were these issues. We have the physical server in the office so it should work fine. Your sentence makes it sound like the issues are connected to the server. We could rephrase it like this "One downside is the need for a physical server to use BBB.
Especially at the beginning, we also had some problems with interruptions and slowly running meetings."

Reply:
Okey. But it is important to say that the biggest part of that problems were fixed